src/main/docker/Dockerfile.jre

```diff
@@ -21,13 +21,17 @@ FROM ${DOCKER_HUB_PUBLIC}/cafapi/opensuse-base:2
 # Refresh the OS repositories and install OpenJDK 17 Runtime Environment
 RUN zypper -n refresh && \
     zypper -n update && \
-    zypper -n install java-17-openjdk && \
+    zypper -n install java-17-openjdk mozilla-nss-tools && \
```

Comment:
Don't install things we don't want in the final image.

Use an interim image and then copy the results in like we do for `gosu` in the base image.